kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -206,12 +206,6 @@
         this.Kolibri.resources.ContentNodeResource.unCacheModel(this.id);
```

Comment:
Left this as is, as couldn't find a convenient place to reference the `id` inside the startTracking action.